components/Elevation/src/UIView+MaterialElevationResponding.h

```diff
@@ -33,6 +33,14 @@
  */
 @property(nonatomic, assign, readonly) CGFloat mdc_baseElevation;
 
+/**
+ Returns the sum of the view's @c mdc_currentElevation with the @c mdc_currentElevation of its
```

Comment:
Nit: "Returns @c mdc_currentElevation plus @c mdc_baseElevation."

Reply:
this is written in the next line, do you still want me to change it?

Reply:
talked offline, resolving.